tests/integration/relationships/has-many-test.js

```diff
@@ -1528,6 +1528,25 @@ test("Polymorphic relationships with a hasMany is set up correctly on both sides
   assert.equal(get(email, 'posts.length'), 1, "The inverse has many is set up correctly on the email side.");
 });
 
+test("Can push a polymorphic belongsTo, the relationship is set up correctly on both sides", function(assert) {
+  assert.expect(2);
+
+  Comment.reopen({
+    message: DS.belongsTo('message', { polymorphic: true, async: false })
```

Comment:
is the intent for this to be a reflexive polymorphic relationship? If not, I suspect you meant to set this to `belongsTo('post'`

Reply:
(If you do intent a reflexive polymorphic relationship, then the test code below would need changes)